kqueen_ui/blueprints/ui/templates/ui/partial/selectform.html

```diff
@@ -43,6 +44,33 @@
   $.validator.addMethod("yamlfile", validateYamlFileField, $.validator.format("Test validation"));
   $.validator.addMethod(
     "cidr", validateFieldIsIpCidr, $.validator.format("Value doesn't match CIDR format."));
+  $.validator.addMethod('uuid', validateFieldIsUuid, 'Entered value is not a valid UUID.');
+
+  // Insert icon with link to the documentation
+  var docLink = `
+    <a href="http://kqueen.readthedocs.io/en/latest/kqueen.html#provision-a-kubernetes-cluster"
+       target="_blank">
+      <span class="simple-svg" data-icon="fa-solid:book" data-inline="false"></span> Documentation
+    </a>`;
+  $(".page-header h2").append(docLink);
+
+  // Show popover on icon hover and keep opened while hovering the popover itself
+  $(".field_help").popover({
+    placement: "auto right", trigger: "manual", animation: false
+  }).on('mouseenter', function () {
+    var _this = this;
+    $(this).popover('show');
+    $('.popover').on('mouseleave', function () {
+      $(_this).popover('hide');
+    });
+  }).on('mouseleave', function () {
+    var _this = this;
+    setTimeout(function () {
```

Comment:
Please add a comment why the timeout is needed here. E.g. that it provides Firefox support and it gives some time for popover to come out